app/models/project_step.rb

```diff
@@ -320,6 +320,10 @@ def pending_days_shifted
     0
   end
 
+  def pending_duration_change?
+    is_finalized? ? scheduled_duration_days_changed? : false
```

Comment:
You can replace this with `is_finalized? && schedule_duration_days_changed?`

As a rule of thumb, you shouldn't need the ternary operator if your goal is to return a boolean.